src/test/java/org/spongepowered/common/launch/mixin/MixinSpongeItemStackSnapshot.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.launch.mixin;
+
+import org.spongepowered.api.item.inventory.ItemStack;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.common.item.inventory.SpongeItemStackSnapshot;
+import org.spongepowered.common.item.inventory.util.ItemStackUtil;
+
+@Mixin(value = SpongeItemStackSnapshot.class, remap = false)
+public abstract class MixinSpongeItemStackSnapshot implements MojangTranslatable {
```

Comment:
Why does this have to be loaded in launch? ItemStackSnapshots are usually loaded well into the game initialization phase.

Reply:
I'm probably missing something. These mixins should only be applied during tests and not shipped in production (Right ?).

Reply:
They are in `src/test/java`.

> However for simplicity sake I would move them to src/main/java.

I'm not sure. The mojang translation is not something we need to expose in the API, it should only be used during tests to reveal inconsitencies.

Reply:
oh I see, I didn't notice the source directory for these mixins, thought you were mixing into them in production and was questioning on that understanding hahah. These are fine, as long as they stay in the test sources.